setup/Product.wxs

```diff
@@ -229,9 +198,6 @@
       <Directory Id="ProgramMenuFolder">
         <Directory Id="ApplicationProgramsFolder" Name="$(var.ProductName)"/>
       </Directory>
-      <Directory Id="DesktopFolder" Name="Desktop">
```

Comment:
I get this:
```
  C:\installer\setup\Product.wxs(98): error LGHT0094: Unresolved reference to symbol 'Directory:DesktopFolder' in section 'Product:*'. [C:\installer\setup\setup.msbuild]
```
The `Directory` element still needs to be kept, like:

```
<Directory Id="DesktopFolder" Name="Desktop"/>
```

Reply:
done